packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/ClientProviders/ClientProviderTestsUtils.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Generator.CSharp.Primitives;
+using Microsoft.Generator.CSharp.Providers;
+using NUnit.Framework;
+
+namespace Microsoft.Generator.CSharp.ClientModel.Tests.Providers.ClientProviders
+{
+    public static class ClientProviderTestsUtils
```

Comment:
This separate file feels unnecessary. Can we just define this in ClientProviderTests?

Reply:
If we do need this, can we follow the existing pattern of suffixing as Helpers rather than Utils.

Reply:
Sure let me put them in ClientProviderTests.
Do you think we should have something globally to assert these things? so that we do not have to define them in each test file

Reply:
If it would be helpful for other tests, then sure. Generally, we have just been creating a new CSharpType and then using the Equals override for these assertions.

Reply:
>  we have just been creating a new CSharpType and then using the Equals override for these assertions.

this is new. Let me follow the same pattern if there is already one.